utils/scripts/install-el8.sh

```diff
@@ -45,22 +45,37 @@ dnf --nodocs install \
     Lmod \
     lz4-devel \
     make \
+    meson \
```

Comment:
Can you try this PR with no changes made to this this?

Reply:
Current push has all changes to that file reverted.